resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/VolcanoTestsSuite.scala

```diff
@@ -201,4 +319,21 @@ private[spark] object VolcanoTestsSuite extends SparkFunSuite {
     getClass.getResource("/volcano/disable-queue0-enable-queue1.yml").getFile
   ).getAbsolutePath
   val GROUP_PREFIX = "volcano-test" + UUID.randomUUID().toString.replaceAll("-", "")
+  val VOLCANO_PRIORITY_YAML
+    = new File(getClass.getResource("/volcano/priorityClasses.yml").getFile).getAbsolutePath
+  val HIGH_PRIORITY_DRIVER_TEMPLATE = new File(
+    getClass.getResource("/volcano/high-priority-driver-template.yml").getFile
+  ).getAbsolutePath
+  val MEDIUM_PRIORITY_DRIVER_TEMPLATE = new File(
+    getClass.getResource("/volcano/medium-priority-driver-template.yml").getFile
+  ).getAbsolutePath
+  val LOW_PRIORITY_DRIVER_TEMPLATE = new File(
+    getClass.getResource("/volcano/low-priority-driver-template.yml").getFile
+  ).getAbsolutePath
```

Comment:
Hm, where did you use `LOW_PRIORITY_DRIVER_TEMPLATE`, `MEDIUM_PRIORITY_DRIVER_TEMPLATE` and `HIGH_PRIORITY_DRIVER_TEMPLATE`?

Reply:
ah, this can be cleanup now, will address